### ISSUE TITLE
User is not able to scroll the "View your page" section of "Printing Testing" with any keyboard key: A11y_WinForms.NETRuntime_PrintingTesting_Keyboard.

### ISSUE THREAD

**Issue by shindepra** [dotnet/winforms#13652]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."

### GitHub Tags
#A11ySev2; #A11yTCS; #WCAG2.1.1; #Win32; #DesktopApp; #Win11; #WinForms .NET Runtime; #BM_WinForms.NETRuntime_Win32_June2025; #keyboard; #A11yMAS

### Environment Details
Application Name: WinForms Runtime
Microsoft Windows 11 Enterprise (24H2) Build on 26100.4061

### Repro Steps

1. Launch the "Accessibility core app".
2. Main Form screen would appear.
3. TAB to "PrintingTesting:Testing button and hit ENTER key.
4. TAB till "View your page" section.
5. Observe that User is not able to scroll the "View your page" section of "Printing Testing" with any keyboard key.

### Actual Result
User is not able to scroll the "View your page" section of "Printing Testing" with any keyboard key.

### Expected Result
"View your page" section of "Printing Testing" should be and user should be able to scroll the section using arrow keys 

### User Impact
Keyboard user will not be able to scroll the "View your page" section of "Printing Testing" with any keyboard key and can not be able to get the data which is available in the View you page section.

### Attachment

https://github.com/user-attachments/assets/7d141b7c-1cde-4e50-881f-11f69b255fb6

**Comment by Olina-Zhang** [comment]
Dup of GH issue: https://github.com/dotnet/winforms/issues/11452, can scroll "View your page" section using **CTRL + Arrow key**

**Comment by merriemcgaw** [comment]
Thanks @Olina-Zhang! 
@shindepra please use the Ctrl+Arrow key to navigate up and down the page, and then we will close this issue.
